### DISCUSSION TITLE
NFBS dataset conversion

### DISCUSSION THREAD

**Discussion by mehnaz1985** [MIC-DKFZ/nnUNet#1123] [General]
if I use the repository and apply on an external dataset, whether I need to commit a conversion.py file for that particular dataset? or if any one have idea about NFBS dataset conversion for nnunet? please let me know. we can work together.

**Comment by FabianIsensee** [comment]
Hey you can convert the dataset using any script you like. It doesn't have to be in the nnU-net repository. If the dataset is publicly available and has good quality then I would accept a PR for a dataset conversion file
